dynamic_bitset.html

```diff
@@ -197,10 +198,13 @@ <h3><a id ="synopsis">Synopsis</a></h3>
     dynamic_bitset <a href="#op-sl">operator&lt;&lt;</a>(size_type n) const;
     dynamic_bitset <a href="#op-sr">operator&gt;&gt;</a>(size_type n) const;
 
+    dynamic_bitset&amp; <a href="#set3">set</a>(size_type n, size_type len, bool val = true);
```

Comment:
This has led to ambiguity.  See https://github.com/rdkit/rdkit/issues/2128.  We need to resolve this.  I will open an issue.

Reply:
This needs to be resolved by Nov 7 or it will be reverted for the Boost 1.69.0 beta.